runtime/browser/cdn/browser-cdn-loader.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const Loader = require('../../loader.js');
+const particle = require("../../particle.js");
+const DomParticle = require("../../dom-particle.js");
+
+module.exports = class BrowserLoader extends Loader {
+  constructor(urlMap) {
```

Comment:
The testing aspect is important, since @shans just ran into a case where we may need to load `schema.org` URLs during tests. Being able to map a URL to a local file path is useful there, so I think we should just push the map down into the base loader for now.